test/sanitizer_suppressions/ubsan

```diff
@@ -78,3 +78,7 @@ implicit-signed-integer-truncation:torcontrol.cpp
 implicit-unsigned-integer-truncation:crypto/*
 implicit-unsigned-integer-truncation:leveldb/*
 implicit-integer-sign-change:crc32c/*
+
+implicit-signed-integer-truncation:script/interpreter.cpp
```

Comment:
Man, is ubsan really this overactive? I see that a bunch of other exceptions in this file already exist, suggesting that it is, but it scares me to suppress things without very careful review of exactly what we're suppressing, line-by-line. I'm surprised we do exceptions for entire files at a time. (Are these specific checks just broken in ubsan?)

Reply:
Yes they are just broken. chars promoted to ints because C++ is bullshit, then "truncated" back within the same expression, etc.